util/posix_close_normal.c

```diff
@@ -0,0 +1,139 @@
+#include <sys/socket.h>
+
+#include <assert.h>
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "noeintr.h"
+#include "warnp.h"
+
+#include "posix_close.h"
+
+/*
+ * The two files pthread_close_normal.c and pthread_close_pthread.c should be
+ * identical, other than the #define USE_PTHREAD line.
+ */
+#define USE_PTHREAD 0
```

Comment:
I haven't included `pthread_close_pthread.c` in this commit, since I'm sure that I'll need to make a bunch of fixes to `posix_close_normal.c` so I might as well make the copy later.